src/controllers/ModelsController.ts

```diff
@@ -62,6 +62,25 @@ const makeModelsController = (
     });
   });
 
+  ModelsController.getAsync('/advancedSearch', async (req, res) => {
+    //A custom route for complex search since we can't use queryToMongo
+    const query = req.body;
+    const options = { limit: req.query.limit, offset: req.query.offset };
+    //we keep the same code here but use the radiksCollection
+    const cursor = radiksCollection.find(query, options);
+    const results = await cursor.toArray();
+    const total = await cursor.count();
+
+    const fullUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`;
+    const pageLinks = mongo.links(fullUrl.split('?')[0], total);
+
+    res.json({
+      ...pageLinks,
```

Comment:
How can pageLinks be used by the client?

Reply:
We have to make pagination manually then

Reply:
Some return value to make that easy should be good.